core/controllers/collection_editor.py

```diff
@@ -121,7 +122,21 @@ def get(self, collection_id):
             'is_public': rights_manager.is_collection_public(collection_id),
             'can_edit': can_edit,
             'collection_id': collection.id,
-            'title': collection.title
+            'title': collection.title,
+            'SHOW_COLLECTION_NAVIGATION_TAB_MAIN': (
```

Comment:
This won't be needed since ``feconf.SHOW_COLLECTION_NAVIGATION_TAB_MAIN`` isn't needed.